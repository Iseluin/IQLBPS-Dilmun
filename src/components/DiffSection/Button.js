import React from "react";

function Button() {
    return (
        <div className="relative">
            <button type="button" className=" bg-blue-500 justify-center p-5 hover:bg-blue-700 text-white font-bold rounded-full">
                Watch Full Video
            </button>
        </div>
    )
}

export default Button;