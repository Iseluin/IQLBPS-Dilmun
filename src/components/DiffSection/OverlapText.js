import React from "react";

function OverlapText() {
    return (
        <div className=" flex flex-col items-center justify-center text-black bg-opacity-20 pl-10 pr-10 ">
            <h2 className=" text-white text-[20px] sm:text-[40px] ">Check this video made by one of our satisfied customers!</h2>
            <button type="button" className="bg-tertiary font-semibold p-2 m-2 hover:bg-blue-700 text-white font-semibold rounded-lg w-full sm:w-auto ">
                Watch Full Video
            </button>
        </div>
    )
}


export default OverlapText;