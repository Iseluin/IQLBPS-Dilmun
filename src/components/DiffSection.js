import React from "react";
 // import { player } from "video-react"
import Dilmun from "../assets/Dilmun.mp4";
import OverlapText from "./DiffSection/OverlapText"


function DiffSection(){
    return (
            <div className="rounded-md shadow-md bg-primary w-full h-96 relative">
                <video src={Dilmun} autoPlay loop muted className="w-full h-full object-cover opacity-60 drop-shadow-2xl rounded-md absolute">
                <track kind="captions"/>
                </video>
                <div  className=" relative mt-14 pt-24 lg:pt-18">
                <OverlapText />
                </div> 
            </div>
    )
}
export default DiffSection ;