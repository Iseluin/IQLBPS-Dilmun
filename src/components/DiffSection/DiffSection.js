import React from "react";
import Dilmun from "../../assets/Dilmun.mp4";
import OverlapText from "./OverlapText"

function DiffSection(){

    return (
            <div className=" rounded-md shadow-md bg-primary w-full h-96 relative">
                <video src={Dilmun} autoPlay loop muted className="w-full h-full object-cover opacity-60 drop-shadow-2xl rounded-md absolute">
                    <track kind="captions"/>
                </video> 
                <div  className=" relative mt-14 pt-14 lg:pt-12 ">
                <OverlapText />
                </div> 
            </div>
    )
}
export default DiffSection ;