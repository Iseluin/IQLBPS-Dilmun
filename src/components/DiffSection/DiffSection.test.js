import React from "react";
import DiffSection from "../DiffSection";
import OverlapText from "./OverlapText";
import Button from "./Button";
import renderer from 'react-test-renderer'
//import { renderer } from "@testing-library/react";

test('DiffSection renders without crashing ', () => {
    const tree = renderer.create(
    <DiffSection>
      <OverlapText>
        <Button />
      </OverlapText>
    </DiffSection>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  
  });

//const Diff = require("../DiffSection");
// test("DiffSection renders without crashing " , () => {
//     const  = render(<DiffSection /> );
//     expect(comp).toBeTruthy();
//     expect(comp).toMatchSnapshot;
// })