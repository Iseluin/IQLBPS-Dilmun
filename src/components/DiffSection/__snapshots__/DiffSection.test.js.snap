// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DiffSection renders without crashing  1`] = `
<div
  className=" rounded-md shadow-md bg-primary w-full h-96 relative"
>
  <video
    autoPlay={true}
    className="w-full h-full object-cover opacity-60 drop-shadow-2xl rounded-md absolute"
    loop={true}
    muted={true}
    src="Dilmun.mp4"
  >
    <track
      kind="captions"
    />
  </video>
  <div
    className=" relative pt-14 lg:pt-12 "
  >
    <div
      className=" flex flex-col items-center justify-center text-black bg-opacity-20 pl-10 pr-10 "
    >
      <h2
        className="text-center text-white text-[16px] sm:text-[32px] "
      >
        Check out this video made by one of our satisfied customers!
      </h2>
      <button
        className="bg-tertiary font-semibold p-2 m-2 hover:shadow-lg text-white font-semibold rounded-lg  sm:w-auto "
        onClick={[Function]}
        type="button"
      >
        Watch Full Video
      </button>
    </div>
  </div>
</div>
`;
